pom.xml

```diff
@@ -9,7 +9,7 @@
     <version>1.0-SNAPSHOT</version>
 
     <properties>
-        <jdk.version>11</jdk.version>
+        <jdk.version>8</jdk.version>
```

Comment:
better to install at least 11-th version of java (now you can leave it as it is)